presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -139,6 +140,7 @@ public PlanOptimizers(
             FeaturesConfig featuresConfig,
             MBeanExporter exporter,
             StatsCalculator statsCalculator,
+            @ForReplicatedJoin StatsCalculator replicatedJoinStatsCalculator,
```

Comment:
`@Optimistic`, `@Pessimistic`. Rename variables to `optimisticStatsCalculator`, `pessimisticStatsCalculator`

Reply:
There is now just `StatsCalculators` object composed of both calculators (probabilistic and upper estimate)